Disable chrony daemon from acting as server.

adelton opened the pull request:
#### Description:

- Disable chrony daemon from acting as server.

#### Rationale:

- Minimizing the exposure of the server functionality of the chrony daemon diminishes the attack space.

adelton commented:
@stevegrubb, could you please provide `identifiers`, `references`, and into what section in `ospp.profile` the rule should fall?

stevegrubb commented:
> @stevegrubb, could you please provide `identifiers`, `references`, and into what section in `ospp.profile` the rule should fall?

This chrony rule falls under FMT_SMF_EXT.1 for its reference. Identifiers are issued by Shawn. He should have some numbers that can be assigned. As for what section, it should be under the services section.

shawndwells commented:
> > @stevegrubb, could you please provide `identifiers`, `references`, and into what section in `ospp.profile` the rule should fall?
> 
> This chrony rule falls under FMT_SMF_EXT.1 for its reference. Identifiers are issued by Shawn. He should have some numbers that can be assigned. As for what section, it should be under the services section.

Grab any available CCE from the available list. Suggest picking one from the middle-ish of the list to avoid creating merge conflicts with existing PRs.

jan-cerny commented:
@adelton The list of available CCEs is in https://github.com/ComplianceAsCode/content/blob/master/shared/references/cce-redhat-avail.txt In your patch, you can remove one number from this file and add the  number to `cce@rhel8:` in `rule.yml`.

adelton commented:
Thanks you all. Updated per recommendation, rebased on master -> d57dd050792c6fb4461796c2f0898a5502984911. @stevegrubb, not sure if I managed to put it exactly where you envisioned it to be.

adelton commented:
I could use `replace_or_append` in the remediation script (similar to https://github.com/ComplianceAsCode/content/pull/4449) if that's viewed as the preferred approach.

yuumasato commented:
@adelton I think we can stick with `replace_or_append` for now, it will be reworked by @cipherboy, and this way it can be easily tracked.

yuumasato commented on linux_os/guide/services/ntp/chronyd_client_only/rule.yml:
```diff
@@ -0,0 +1,30 @@
+
+rationale: |-
+    Minimizing the exposure of the server functionality of the chrony
+    daemon diminishes the attack space.
```
nitpick: I think a more broadly used term is `attack surface`.

adelton replied:
Fixed in 430c2d727d117cacff5ff0ebd50df78cc3671251.

adelton commented:
> we can stick with `replace_or_append`

Done in 430c2d727d117cacff5ff0ebd50df78cc3671251. Also rebased on master.

jan-cerny commented:
@adelton please resolve the conflicts

adelton commented:
Rebased on master -> 3e9ddff1050d29a7b4d84046fb298d4827d58429.

jan-cerny commented:
Thank you.
